packages/loader/container-loader/src/loader.ts

```diff
@@ -430,37 +409,6 @@ export class Loader implements IHostLoader {
 		);
```

Comment:
i would like to see the request this function removed as well. there already exists a replacement, requestResolvedObjectFromContainer, and this function is even worse to use without caching, as it makes it impossible to get the container instance.

Reply:
can be a follow up PR, but should land in the same major